### DISCUSSION TITLE
cannot add mega disk

### DISCUSSION THREAD

**Discussion by wieniawski-lai** [AlistGo/alist#2809] [Q&A]
I am using alist version3.6.0, and host OS is win10 home.

I am trying to add mega net-disk to alist. I entered the mega account, password and etc. Then I click save, but no response.

Then in Storage tab, I found error:
状态: 
Post "https://g.api.mega.co.nz/cs?id=3152391081": dial tcp 173.231.12.107:443: i/o timeout

I am sure I can access mega in my web-browser.

Does any one knows how to resolve that

**Comment by Skylarzz** [comment]
You need to set up a VPN for Alist, FYI https://github.com/alist-org/alist/discussions/4363
